files/es/web/api/batterymanager/onchargingchange/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: BatteryManager.onchargingchange
+slug: Web/API/BatteryManager/onchargingchange
+translation_of: Web/API/BatteryManager/onchargingchange
+browser-compat: api.BatteryManager.onchargingchange
+---
+{{deprecated_header}}
+
+{{APIRef("Battery API")}}
+
+Especifica un evento que escucha para recibir eventos {{event("chargingchange")}}. Estos eventos se producen cuando se actualiza el estado de la batería {{domxref("BatteryManager.charging", "charging")}}.
+
+## Sintaxis
+
+```js
+battery.onchargingchange = funcRef
+```
+
+Donde `battery` es un objeto {{domxref("BatteryManager")}}, y `funcRef` es una función para llamar cuando se produce el evento
+{{event("chargingchange")}}.
+
+## Ejemplo
+
+### Contenido HTML
+
+```html
+<div id="nivel">(nivel de batería desconocido)</div>
+<div id="tiempodeCarga">(tiempo de carga desconocido)</div>
+```
+
+### Contenido JavaScript
+
+```js
+navigator.getBattery().then(function(battery) {
+
+   battery.onchargingchange = chargingChange();
+
+   function chargingChange() {
+      document.querySelector('#nivel').textContent = battery.level;
+      document.querySelector('#tiempodeCarga').textContent = battery.chargingTime;
+   }
+});
+```
+
+### Resultado
+
+{{EmbedLiveSample('', '100%', 40)}}
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad con navegadores
+
+{{Compat}}
+
+## Ver también
+
+- {{domxref("BatteryManager")}}
+- {{domxref("Navigator.getBattery")}}
```

Comment:
Nitpicking here, but leaving a newline at the end of a file is an MDN standard

```suggestion
- {{domxref("Navigator.getBattery")}}

```

Reply:
Great, thank you. @cw118